cmd/config/internal/commands/cmdlistsetters_test.go

```diff
@@ -469,3 +461,52 @@ openAPI:
 		})
 	}
 }
+
+func TestListSettersSubPackages(t *testing.T) {
```

Comment:
I think we could use some more tests here. At least cover the situation where a subpackage has no setters or substitutions.

Reply:
Added.